services/core-web/src/components/mine/Securities/MineSecurityInfo.js

```diff
@@ -245,11 +248,21 @@ export class MineSecurityInfo extends Component {
             <div>
               <div className="dashboard--cards">
                 <MineDashboardContentCard
-                  title="Total Security Held"
+                  title={
+                    <span>
+                      Total Security Held
+                      <CoreTooltip title="Total Security Held: This is the total amount of all active bonds held on this mine record. If the mine is on a payment schedule, it shows only what has been paid to date. It includes any interest earned." />
```

Comment:
"...This is the total amount of all active bonds held..." I think this should be changed to mention the dollar amount associated with the bonds, it could be read as the number of active bonds instead.